packages/create-cloudflare/src/workers.ts

```diff
@@ -34,6 +34,13 @@ export const runWorkersGenerator = async (args: C3Args) => {
 		args,
 	};
 
+	ctx.args.ts = await processArgument<boolean>(ctx.args, "ts", {
```

Comment:
either `processArgument` should handle assigning the `args.x` value, ie:
```suggestion
	await processArgument<boolean>(ctx.args, "ts", {
```

or the pattern should be to call it with `??=`, ie:
```suggestion
	ctx.args.ts ??= await processArgument<boolean>(ctx.args, "ts", {
```

Reply:
Yeah, agreed this feels a little unwieldy still.

One of the important things `processArgument` does is render the `submit` state of the prompt for the question, so a user would see `yes Typescript` even if they passed the `--ts` flag. That's why `ctx.args.ts ??= await processArgument<boolean>(ctx.args, "ts", {` wouldn't work, since we do need it to evaluate.

I like the first alternative though. I'll make a separate PR to refactor this into an in-place operation instead of returning a value that we re-assign to the same object.